util/etag.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package util
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"io"
+	"os"
+)
+
+// ComputeEtag returns etag for a file at path
+func ComputeEtag(path string) (string, error) {
```

Comment:
this is cause they write the etag themselves so they use md5? or do we control that also